02/app.js

```diff
@@ -1,27 +1,36 @@
 /* rozwiązanie z pętlą for */
-let x = 4;
 const userNumber = parseInt(prompt('Podaj liczbę', 4));
 if (userNumber <= 9 && userNumber > 0) {
-  x = userNumber;
+  for (let i = 1; i <= 9; i++) {
+    console.log(`${userNumber} x ${i} = ${userNumber * i}`);
+  }
 } else {
   alert('Podaj liczbę w zakresie od 1 do 9!');
 }
-for (let i = 1; i <= 9; i++) {
-  console.log(`${x} x ${i} = ${x * i}`);
-}
 
-/* rozwiązanie z pętlą while  */
+// /* rozwiązanie z pętlą while  */
 
 // const a = Number(prompt('Podaj liczbę podstawową'));
 // const b = Number(prompt('Podaj liczbę wykładniczą'));
-// let result = 1;
-// while (result <= b) {
-//   result++;
-//   if (a >= 1 && a <= 9 && b >= 1 && b <= 9) {
+
+// if (
```

Comment:
Jak oddajesz zadanie lepiej nie zakomentowywać ;)
Łatwo można to przeoczyć, bo raczej kod zakomentowany pomijam ;P